concent_api/core/tests/test_core_views.py

```diff
@@ -0,0 +1,74 @@
+import json
+
+from freezegun   import freeze_time
+from django.test import TestCase, Client
+from django.urls import reverse
+
+from core.models import Message, MessageStatus
+
+
+class CoreViewSendTest(TestCase):
+
+    def setUp(self):
+        self.client = Client()
+        self.correct_data = {
+            "type":      "MessageForceReportComputedTask",
+            "timestamp": 1510911047,
+            "message_task_to_compute": {
+                "type":               "MessageTaskToCompute",
+                "timestamp":          1510909200,
+                "task_id":            8,
+                "deadline":           1510915047
+            }
+        }
+        self.public_key = '85cZzVjahnRpUBwm0zlNnqTdYom1LF1P1WNShLg17cmhN2UssnPrCjHKTi5susO3wrr/q07eswumbL82b4HgOw=='
+
+    @freeze_time("2017-11-17 10:00:00")
+    def test_send_should_accept_valid_message(self):
+        assert Message.objects.count()       == 0
+        assert MessageStatus.objects.count() == 0
+
+        response = self.client.post(reverse('core:send'), data = json.dumps(self.correct_data), content_type = 'application/json', HTTP_CONCENT_CLIENT_PUBLIC_KEY = self.public_key)
+
+        self.assertEqual(response.status_code, 202)  # pylint: disable=no-member
+        self.assertEqual(len(Message.objects.all()),       1)
+        self.assertEqual(len(MessageStatus.objects.all()), 1)
```

Comment:
You should check that the content of messages added to the database actually matches what has been sent.